_pages/en_US/get-started.txt

```diff
@@ -76,13 +76,13 @@ There have been reports of a wave of bans being handed out to CFW users by Ninte
       <td style="text-align: center; font-weight: bold;"><a href="homebrew-launcher-(soundhax)">Homebrew Launcher (Soundhax)</a></td>
     </tr><tr>
       <td style="text-align: center; font-weight: bold;">11.4.0</td>
-      <td style="text-align: center; font-weight: bold;">11.11.0</td>
+      <td style="text-align: center; font-weight: bold;">11.12.0</td>
       <td style="text-align: center; font-weight: bold;">Update to latest firmware or use an "All Versions" Method</td>
     </tr>
     <tr>
-      <td style="text-align: center; font-weight: bold;">11.12.0</td>
       <td style="text-align: center; font-weight: bold;">11.13.0</td>
-      <td style="text-align: center; font-weight: bold;"><a href="seedminer">Seedminer</a></td>
+      <td style="text-align: center; font-weight: bold;">11.13.0</td>
+      <td style="text-align: center; font-weight: bold;"><a href="homebrew-launcher-(browserhax-2020)">BrowserHax 2020</a></td>
```

Comment:
In line of the rest of the guide, it should be labelled `Homebrew Launcher (<whatever we end up calling this>)`

Reply:
pending further conversation

Reply:
I agree with lily. It feels inconsistent with the 9.2.0-11.2.0 soundhax options